src/consent.js

```diff
@@ -95,6 +96,34 @@ export function getConsentMetadata(element, policyId = 'default') {
   );
 }
 
+/**
+ * @param {!Element} element
+ * @param {?string=} opt_policyId
+ * @return {!Promise<!JsonObject>}
+ *    - consentMetadata
+ *    - consentString
+ *    - consentPolicyState
+ */
+export function collectConsents(element, opt_policyId) {
```

Comment:
nit: `policyId = 'default'`

Reply:
In this case the argument could be both`null` or `undefined` so default params won't work (they're only set for `undefined`).

![image](https://user-images.githubusercontent.com/254946/103832731-db6f7100-5033-11eb-8093-ce40a2e15f75.png)

Reply:
Interesting! Thanks for the tidbit